config/sync/core.entity_form_display.paragraph.instructions.default.yml

```diff
@@ -0,0 +1,29 @@
+uuid: 81174690-ec9d-439b-98bd-fbf45b42a1fc
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.paragraph.instructions.field_p_i_instruction
+    - paragraphs.paragraphs_type.instructions
+  module:
+    - paragraphs
+id: paragraph.instructions.default
+targetEntityType: paragraph
+bundle: instructions
+mode: default
+content:
+  field_p_i_instruction:
+    type: entity_reference_paragraphs
+    weight: 0
+    settings:
+      title: Paragraph
+      title_plural: Paragraphs
+      edit_mode: open
+      add_mode: dropdown
+      form_display_mode: default
+      default_paragraph_type: ''
+    third_party_settings: {  }
+    region: content
+hidden:
+  created: true
+  status: true
```

Comment:
Ensure this paragraph and the method paragraph below use the stable paragraph type and also ensure edit mode is "Closed" not open

Reply:
done